yt_dlp/extractor/bilibili.py

```diff
@@ -433,41 +433,56 @@ def _real_extract(self, url):
 
         if '您所在的地区无法观看本片' in webpage:
             raise GeoRestrictedError('This video is restricted')
-        elif ('开通大会员观看' in webpage and '__playinfo__' not in webpage
-                or '正在观看预览，大会员免费看全片' in webpage):
+        elif ('成为大会员免费看' in webpage
+              or '正在观看预览，大会员免费看全片' in webpage
+              or '开通大会员观看' in webpage):
```

Comment:
Maybe we should remove checks for `开通大会员观看` as I got this part even I am a "Big" account. HTML snippet with this string:

```
"payment":{"price":"0.0","tip":"大会员专享观看特权哦~","vip_promotion":"开通大会员观看","vip_first_promotion":"","vip_discount":1,"pay_type":{"allow_ticket":false,"allow_vip_discount":false},"vip_price":"0"}
```

Reply:
> Maybe we should remove checks for `开通大会员观看` as I got this part even I am a "Big" account. HTML snippet with this string:
> 
> ```
> "payment":{"price":"0.0","tip":"大会员专享观看特权哦~","vip_promotion":"开通大会员观看","vip_first_promotion":"","vip_discount":1,"pay_type":{"allow_ticket":false,"allow_vip_discount":false},"vip_price":"0"}
> ```

This string will also appear in the first episode of some permium dramas, but in fact the first episode may can be watched without a "Big" account.

Reply:
I changed it so that the extractor would only `raise_login_required` for that string if no formats are found. Have you had a chance to test the changes @GD-Slime ?

Reply:
> raise_login_required

I tested it. But in fact, when you should not have permission to watch a certain bangumi, the `https://api.bilibili.com/pgc/player/web/v2/playurl` api will directly return a special value without any format in it. I fixed it and will commit later.